src/internal/m365/graph/errors_test.go

```diff
@@ -271,6 +266,55 @@ func (suite *GraphErrorsUnitSuite) TestIsErrInvalidDelta() {
 	}
 }
 
+func (suite *GraphErrorsUnitSuite) TestIsErrDeltaNotSupported() {
+	table := []struct {
+		name   string
+		err    error
+		expect assert.BoolAssertionFunc
+	}{
+		{
+			name:   "nil",
+			err:    nil,
+			expect: assert.False,
+		},
+		{
+			name:   "non-matching",
+			err:    assert.AnError,
+			expect: assert.False,
+		},
+		{
+			name:   "as",
```

Comment:
should we be checking if a wrapped error is handled correctly?

Reply:
That feels a bit extraneous, given how well tested error wrapping systems are already.

Reply:
we did previously had a bug where some of code was expecting the error type itself (i.e. direct type-assertion to error type) and not a wrapped version of the error (i.e. obtain error via `errors.As` or `errors.Is`)

Reply:
I do recall that case.  I'm still going to punt here, because the topic is getting at the broader design of error handling here, for which we already have a ticket, and the work can be left to that.  In the meantime I'll see about a little bump on the side.